lib/matplotlib/dates.py

```diff
@@ -7,8 +7,10 @@
 Matplotlib date format
 ----------------------
 Matplotlib represents dates using floating point numbers specifying the number
-of days since 0001-01-01 UTC, plus 1.  For example, 0001-01-01, 06:00 is 1.25,
-not 0.25. Values < 1, i.e. dates before 0001-01-01 UTC, are not supported.
+of days since a defaul epoch of 0001-01-01 UTC, plus 1. For example,
```

Comment:
Let's get rid of this horrible "plus 1" business, so that at last we are really talking about time in days since the epoch.  (Matlab now fudges this: "A serial date number represents the whole and fractional number of days from a fixed, preset date (January 0, 0000) in the proleptic ISO calendar."  Who knew there was a January 0?  We don't have to be quite as crazy; ISO 8601 does include a year 0000, so our old default epoch is a valid ISO date, even though datetime.date doesn't handle it.

Reply:
OK, plus one is gone!